pages/authCallbackOAuth2/authCallbackOAuth2.js

```diff
@@ -52,6 +52,7 @@ router.get('/', function (req, res, next) {
       identity.name || identity.email, // name (use email if name is not present)
       identity.sub, // uin
```

Comment:
I think we should not provide the UIN from Google anymore?

Reply:
Or maybe we do want this, actually. But we should add a warning to the institution auth setup page to warn that enabling Google auth will overwrite UINs if you use it in conjunction with SAML? You basically should never turn on both Google and SAML for an institution, I think.

Reply:
Yeah, I'm really not sure about this. My gut tells me that Azure and Google (OAuth) should basically behave the same, and since Azure doesn't include a UIN, neither should Google? I'd be happy to just remove this.

Reply:
I guess I'm thinking that if an institution is using Google as their primary auth source then we _do_ want to use the UIN from it, to support changes in the UID. Ideally we'd also get an equivalent stable identifier from Azure. At the moment, if an institution uses Azure as it's auth source, we can't support UID renames.

Reply:
Ah, that's fair. In the interest of shipping these changes ASAP, I won't make that change here, but I did open a new issue to track that: #6164. Does that sound good to you?

Reply:
Perfect